src/main/java/uk/ac/cam/cl/dtg/segue/api/Constants.java

```diff
@@ -386,11 +386,14 @@ public enum TimeInterval {
         TimeInterval(final int years, final int months, final int days, final int hours, final int minutes, final double seconds) {
             this.interval = new PGInterval(years, months, days, hours, minutes, seconds);
         }
+
         public PGInterval getPGInterval() {
             return this.interval;
         }
     }
 
+
+
```

Comment:
Formatting again. Is there a reason for the extra empty lines or are they just left over from something else?